src/main/java/core/basesyntax/Bulldozer.java

```diff
@@ -0,0 +1,13 @@
+package core.basesyntax;
+
+public class Bulldozer extends Machine {
+    @Override
+    public void doWork() {
+        System.out.println("Bulldozer: do work");

```

Comment:
The message in the `doWork` method should be more informative and indicate that the Bulldozer is working. Consider specifying the action as well as the type of Machine, for example: 'Bulldozer started its work.'